tests/security/audit/auditd.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Controlling the Auditd daemon as root by "systemctl" to verify it can work
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772 tc#1768549
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # Make sure audit packages are installed
+    zypper_call('in audit libaudit1');
```

Comment:
Please double confirm the packages are installed by default. 
I remembered they are installed, so we don't need install them any more [not 100% sure]

Reply:
zypper_call('audit') if script_run('rpm -q audit');

Reply:
Added package checking.